source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ADCIOSAdaptiveHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ADCIOSAdaptiveHostConfig:NSObject
+
+- (CGSize) getImageSize:(std::shared_ptr<Image>) imgBlock;
+
+- (UIColor* ) getTextBlockColor:(std::shared_ptr<AdaptiveCards::TextBlock>) txtBlock;
+
+- (int) getTextBlockTexSize:(std::shared_ptr<TextBlock>)txtBlock;
```

Comment:
>getTextBlockTexSize [](start = 8, length = 19)

A lot of the HostConfig properties apply to more than one element. Maybe this should take an OBJC textview instead of the object model textBlock? That way you can reuse this for fact, and any other element that needs to have its text style changed